README.md

```diff
@@ -54,7 +54,7 @@ Trino supports [reproducible builds](https://reproducible-builds.org) as of vers
 ## Build requirements
 
 * Mac OS X or Linux
-* Java 22.0.0+, 64-bit
+* Java 23.0.0+, 64-bit
```

Comment:
maybe ?? 

```suggestion
* Java 23.0.1+, 64-bit
```

Reply:
The current value is ok. The minimum required version for building and running is still 23.0.0. Unless we find a critical issue in a given minor version, there's no reason to bump the minimum requirement.

Reply:
Yep, that's my reasoning as well.